distributed/utils.py

```diff
@@ -49,7 +52,47 @@ def get_num_params(model: torch.nn.Module, exclude_embedding: bool = False) -> i
     num_params = sum(p.numel() for p in model.parameters())
     if exclude_embedding:
         num_params -= model.tok_embeddings.weight.numel()
-    return num_params
+    readable_num_params = format_model_params(num_params)
+    return readable_num_params
+
+
+def get_stage_size(stage):
```

Comment:
nit: can we name it "get_module_size"? It seems to me that the function can be generally applied to `nn.Module` than just a stage. Also, we can type the input arg.

Reply:
added typiing and changed name.